Dockerfile

```diff
@@ -1,22 +1,20 @@
 FROM node:lts
 
-RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 ARG NODE_ENV
 ENV NODE_ENV $NODE_ENV
 
-COPY install/package.json /usr/src/app/package.json
+COPY install/package.json package.json
 
 RUN npm install --only=prod && \
     npm cache clean --force
     
-COPY . /usr/src/app
+COPY . .
```

Comment:
Can you revert these lines? I believe in this specific case, it is better to be explicit as to where the files are copied to

Reply:
@julianlam Thanks for your comments. Docker has WORKDIR instruction (WORKDIR /usr/src/app). And we can use it shorter path.

Reply:
Yes, that is correct (I had a Docker expert advise me on the PR, as I am not very familiar). However, I still feel that being more explicit is best, otherwise if `WORKDIR` is removed, then the files will be installed to `/` which is not recommended.

Reply:
Ok 👍, for what reasons can `WORKDIR` be removed?

Reply:
The intention is that if these instructions are forked and modified, then the behaviour will be unpredictable because the assumption that the cwd is `WORKDIR` is precisely that, an assumption